addons/html_editor/static/tests/list/list_font_size.test.js

```diff
@@ -8,12 +8,45 @@ import {
 } from "../_helpers/user_actions";
 import { execCommand } from "../_helpers/userCommands";
 
+function getMarkerWidth(li) {
+    const targetUl = li.parentElement.cloneNode();
+    const targetLi = li.cloneNode(true);
+    const tempEl = document.createElement("temp-el");
+    // we need some text as empty `li` will not give proper result later
+    targetLi.textContent = ".";
+    // for `UL` when `listStylePosition = "inside"` there is always
+    // some extra space, we try to reduce the size by 20%.
+    const reduceBy = targetUl.nodeName === "UL" ? 0.8 : 1;
+
+    targetUl.style.visibility = "hidden";
+    targetUl.style.position = "absolute";
+    const isRTL = targetUl.getAttribute("dir") === "rtl";
+    if (isRTL) {
+        targetUl.style.right = "0";
+    }
+
+    targetLi.prepend(tempEl);
+    targetUl.appendChild(targetLi);
+    document.body.append(targetUl);
+
+    const posInside = tempEl.getBoundingClientRect().left;
+    targetLi.style.listStylePosition = "inside";
+    const posOutside = tempEl.getBoundingClientRect().left;
+
+    targetUl.remove();
+    return Math.floor(Math.abs(posOutside - posInside) * reduceBy);
+}
+
```

Comment:
Why is this function duplicated? Can't you make it into a util and import it in both the plugin and the test since it doesn't use `this`?